toto/models/layout.py

```diff
@@ -104,6 +104,15 @@ def read(data):
         keys=data.get("keys"), expires=data.get("expires"),
         signatures=data.get("signatures"))
 
+  def import_step_metadata_from_files_as_dict(self):
```

Comment:
Is this function going to be used by other modules? If so, maybe we need a more elaborate docstring. Otherwise, we may want to prepend this function name with a _ to hint that it's a private function.